Tensile/Configs/rocblas_hpa_bfloat16_hip_single_kernel.yaml

```diff
@@ -8,17 +8,26 @@ GlobalParameters:
   EnqueuesPerSync: 1
   SyncsPerBenchmark: 1
   LibraryPrintDebug: False
-  NumElementsToValidate: 0
+  NumElementsToValidate: 8
   ValidationMaxToPrint: 4
   ValidationPrintValids: False
   ShortNames: False
   MergeFiles: True
   Platform: 0
   Device: 0
   KernelTime: True
-  PinClocks: True
+  PinClocks: False
```

Comment:
Did you mean to change this?
Perhaps could also change SleepPercent to 0 if this intended to be file for tuning?

Reply:
I was aiming to make it simpler. I am using this .yaml file to generate the simplest kernel and test I can think of for initial testing. 

The other _single_kernel.yaml files generate a kernel that gives good performance for large size. This file should change to a generating a kernel for large size as the bfloat16 implementation continues, and as other .yaml files are added for testing and tuning.